app/ts/components/requirements.tsx

```diff
@@ -123,11 +145,11 @@ export const Immutable = ( { checkBoxesArray } : { checkBoxesArray: OptionalSign
 		<div style = 'padding-top: 30px; padding-bottom: 30px; align-items: center; display: grid; width: 100%'>
 			{ checkBoxes.immutable ? <>
 				<p class = 'status-green'>
-					{`IMMUTABLE for about ${ dateDiff } (until ${ dateString })` }
+					{`Almost IMMUTABLE* for about ${ dateDiff } (until ${ dateString })` }
```

Comment:
Thoughts on moving the `*` to `Almost` instead of `IMMUTABLE`?  I can see it making sense on either, and I'm happy with whichever one you choose.
```suggestion
					{`Almost* IMMUTABLE for about ${ dateDiff } (until ${ dateString })` }
```

Reply:
yaeh, I think its more accurate being next to almost. I move it there